PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Feature or Problem
+Briefly describe the reason for this pull request: the feature being added or problem being solved.
+
+# Related Issues
+Link to any issues or correlated pull requests that are related to this PR. For example, if this PR fixes an issue, link to that issue here.
+
+# Release Information
+Clearly state the target release for this code. If there isn't a specific target version, you can state the `next` release, etc.
+
+# Consumer Impact
+Indicate the impact, if any, this change will have on other consumers, dependencies, or dependents. In other words, the "blast radius" of the impact of this change
+and what steps related projects may need to take in response to this.
+
+# Testing
+Declare the testing information for this pull request
+
+## Platforms
+Identify the platforms on which this code was tested (include both OS and CPU architecture)
```

Comment:
Change to checklist:

<!-- Please check what applies. Note that these are not hard requirements but serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin

Reply:
I like how nixpkgs does these: https://raw.githubusercontent.com/NixOS/nixpkgs/master/.github/PULL_REQUEST_TEMPLATE.md